NEWS.md

```diff
@@ -488,3 +510,5 @@ changelog for details of the default gems or bundled gems.
 [Feature #19965]: https://bugs.ruby-lang.org/issues/19965
 [Feature #20005]: https://bugs.ruby-lang.org/issues/20005
 [Feature #20057]: https://bugs.ruby-lang.org/issues/20057
+[Bug #19293]: https://bugs.ruby-lang.org/issues/19293
+[Feature #18285]: https://bugs.ruby-lang.org/issues/18285
```

Comment:
When removing the NoMethodError change, we would want to remove this line as well.

Reply:
It's better to sort by URLs.